Sources/KSCrashRecordingCore/include/KSBacktrace.h

```diff
@@ -0,0 +1,98 @@
+//
+// KSBacktrace.h
+//
+// Created by Alexander Cohen on 2025-05-27.
+//
+// Copyright (c) 2012 Karl Stenerud. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall remain in place
+// in this source code.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#ifndef HDR_KSBacktrace_h
+#define HDR_KSBacktrace_h
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <pthread.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Captures the backtrace (call stack) for the specified pthread.
+ *
+ * @param thread      The identifier of the pthread whose backtrace should be captured. Must be a valid, non-null
+ * thread.
+ * @param addresses   A pointer to a buffer to receive the backtrace addresses. Must not be NULL.
+ * @param count       The maximum number of addresses to capture. Must be greater than zero.
+ *
+ * @return The number of frames captured and written to @c addresses, or 0 if @c addresses is NULL, @c count is zero, or
+ * an error occurs.
+ *
+ * @discussion This function is not async-signal-safe and therefore must not be called from within a signal handler.
+ *             It may also briefly suspend the target thread while unwinding its stack.
+ */
+
+int ks_captureBacktrace(pthread_t _Nonnull thread, uintptr_t *_Nonnull addresses, int count)
```

Comment:
I think `ksbt_` would follow better established pattern - other modules in KSCrashRecordingCore use module-specific prefixes like `ksdl_`, `ksbic_`, `ksmc_`, etc.

Reply:
done